uaclient/tests/test_cli_reboot_required.py

```diff
@@ -0,0 +1,37 @@
+import mock
+import pytest
+
+from uaclient.cli import main
+
+HELP_OUTPUT = """\
+usage: pro system <command> [flags] reboot-required [-h]
```

Comment:
I'm not sure how to fix this but the `<command>` is misleading here. Preferably it would look like
```suggestion
usage: pro system reboot-required [-h]
```